xml/mdns_device-info_txt.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0"?>
+<!--
+  Fingerprint definitions that are matched against the string values in the
+  TXT record returned by an mDNS endpoint for the '_device-info._tcp.local.'
+  domain name.
```

Comment:
Here and in the other file, it may be helpful to re-mention the fact that you may need to prepend the host name.